src/main/java/cms/gov/madie/terminology/controller/VsacController.java

```diff
@@ -0,0 +1,49 @@
+package cms.gov.madie.terminology.controller;
+
+import java.util.concurrent.ExecutionException;
+
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import cms.gov.madie.terminology.service.VsacService;
+
+import lombok.extern.slf4j.Slf4j;
+import generated.vsac.nlm.nih.gov.RetrieveMultipleValueSetsResponse;
+
+@RestController
+@RequestMapping(path = "/vsac")
+@Slf4j
+public class VsacController {
+
+  private final VsacService vsacService;
+
+  public VsacController(VsacService vsacService) {
+    this.vsacService = vsacService;
+  }
+
+  @GetMapping(
+      path = "/valueSet",
+      consumes = "text/plain",
+      produces = MediaType.APPLICATION_XML_VALUE)
```

Comment:
Would it reasonable to re-parse the XML into JSON for front end consumption? 

I doubt we will have a use case for XML manipulation in the front end so I would like to avoid it where possible.

Actually, perhaps we can return the results as [FHIR ValueSet Resources](https://www.hl7.org/fhir/valueset.html), that would help us avoid having to convert the result to the Resource for execution.

Reply:
@jkotanchik-SB This is done, I just pushed in the changes. Thanks!